CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ Version template:
 # Dynamic Extensions For Alfresco Changelog
 
 ## [2.1.1] - UNRELEASED
+### Changed
+* [#331](https://github.com/xenit-eu/dynamic-extensions-for-alfresco/issues/331) Change `:webscripts` to provide 2 versions, for 50-62 & and 70 respectively
```

Comment:
This comment is what changed internally. Can we reflect here what this means for users of the DE framework? 

### Fixed
_JSONWriterResolution on Alfresco 7_

?